qutip/tests/core/data/test_mathematics.py

```diff
@@ -816,6 +802,18 @@ def test_incorrect_shape_raises(self, op, data_m):
             op(data_m(), 10)
 
 
+class TestExpm(UnaryOpMixin):
+    def op_numpy(self, matrix):
+        return sc.sparse.linalg.expm(matrix)
```

Comment:
I didn't catch this in time, but there's no need to use `scipy.sparse.linalg` here; the matrix is guaranteed to be a Numpy array by the test sutie.